ui/src/app/dashboard/dashboard.component.html

```diff
@@ -1,12 +1,22 @@
 <jm-header [showControls]="false"></jm-header>
-
 <div class="aggregation-container">
-  <jm-total-summary [summary]="aggregationResponse.summary" [statusArray]="statusArray"></jm-total-summary>
+  <span class="title">Total Jobs</span>
+  <mat-select [placeholder]='tfDescMap.get(selectedTimeFrame)'
+              (selectionChange)="onTimeFrameChange($event.value)">
+    <mat-option *ngFor="let timeFrame of timeFrames" [value]="timeFrame">
+      {{tfDescMap.get(timeFrame)}}
+    </mat-option>
+  </mat-select>
+
+  <jm-total-summary [summary]="aggregationResponse.summary"
+                    [statusArray]="statusArray"
```

Comment:
This reminds me of the statusArray issue (where the card renders poorly for >4 statuses). Can you file an issue for this as well?

Reply:
Done.